lib/matplotlib/backends/backend_ps.py

```diff
@@ -1557,12 +1557,14 @@ def xpdf_distill(tmpfile, eps=False, ptype='letter', bbox=None, rotated=False):
         command = [str("ps2pdf"), "-dAutoFilterColorImages#false",
                    "-dAutoFilterGrayImages#false",
                    "-sGrayImageFilter#FlateEncode",
+                   "-dAutoRotatePages=false",
```

Comment:
needs a hash to make windows happy... (see above too)
unfortunately we're not currently testing this codepath on windows CI

Reply:
ooops!